### PR TITLE
Feat/demo

### PR DESCRIPTION
**Pull request by LarsDenBakker**
Adds demo projects for lit-element and lit-element-ts. Shows example of using esbuild plugin.

Fixes a publishing error in esbuild plugin.

### CODE REVIEW

File: demo/projects/lit-element-ts/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@web/demo-lit-element-ts",
+  "version": "0.0.0",
```
Comment (daKmoR) [inline demo/projects/lit-element-ts/package.json:3]: private: true missing?

File: demo/projects/lit-element/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "name": "@web/demo-lit-element",
+  "version": "0.0.0",
+  "description": "Webcomponent lit-element following open-wc recommendations",
+  "license": "MIT",
+  "author": "my-element",
+  "main": "index.js",
+  "module": "index.js",
+  "scripts": {
+    "start": "es-dev-server --app-index demo/index.html --node-resolve --open --watch",
+    "test": "wtr test/**/*.test.js --coverage",
```
Comment (daKmoR) [inline demo/projects/lit-element/package.json:11]: I would prefer to using web-test-runner in our demos and documentation as acronyms are always tough especially for new users

File: demo/projects/lit-element-ts/package.json (hunk 1 above)
Comment (LarsDenBakker) [reply demo/projects/lit-element-ts/package.json:3]: yep

File: demo/projects/lit-element/package.json (hunk 2 above)
Comment (LarsDenBakker) [reply demo/projects/lit-element/package.json]: done

**Pull request merged**